backend/vkt/src/main/java/fi/oph/vkt/model/type/EnrollmentStatus.java

```diff
@@ -0,0 +1,8 @@
+package fi.oph.vkt.model.type;
+
+public enum EnrollmentStatus {
+  PAID,
+  EXPECTING_PAYMENT,
+  QUEUED,
+  CANCELED,
```

Comment:
Voiko olla (ja tarvitseeko enumeroida) tilanne jossa on maksettu ja peruttu?

Reply:
Toi CANCELED itse asiassa kuvaa juuri tätä tilaa, sillä tarvitseeko jonossa olevia ilmoittautumisia pystyä peruuttamaan?

Reply:
Ai ok, jotenkin tätä lukiessani ajattelin että CANCELED on tilanne jossa ilmoittautuminen peruttiin ennen maksamista. Varmaan ihan ok siis näin.